### ISSUE TITLE
Discuss Mindfulness Policy

### ISSUE THREAD

**Issue by DanM3rcurius** [SecretDecks/Documentation#10]
## Goal
Reach a consensus about the content of the policy: Can everyone in the core ACO agree on the terms stated within it?

### Specifications
1. Discuss
2. document results
3. apply edits if needed
4. vote on final approval
5. Publish in SD github repository

### Milestone
0 - Primer

**Comment by DanM3rcurius** [comment]
@SecretDecks/founding-core please leave your feedback about the mindfulness policy here as comments 🙏

**Comment by tamagit80** [comment]
Hello @SecretDecks/founding-core how are you? Hope this message finds you all well!
I'm sorry if I still didn't find some time to introduce myself but working hard on the SD logo, website... and in the main time have my 'private' jobs. I'll find some time today hopefully! By the way... about the MFP doc, there are only 2 points I do not agree completely:
- Love, over fear of change. Why fear of change and not only fear?
- Game B over Game A. Is not very clear, i think we can explain better the concept. Actually I can't get it... "everything is always discussable"?

Hope it can help!
Cheers G.

**Comment by wlstylr** [comment]
@SecretDecks/founding-core 
How about the following to replace "Love, over fear of change"

- Love, over fear
- Fluidity over conformity 

For "Game B over Game A, we may want to link to a video or concise article that can quickly explain this concept to those unfamiliar.

**Comment by DanM3rcurius** [comment]
> Hello @SecretDecks/founding-core how are you? Hope this message finds you all well!
> I'm sorry if I still didn't find some time to introduce myself but working hard on the SD logo, website... and in the main time have my 'private' jobs. I'll find some time today hopefully! 
All good and thanks for the amazing work you've been doing 🙏

By the way... about the MFP doc, there are only 2 points I do not agree completely:
> 
> * Love, over fear of change. Why fear of change and not only fear?
Good point! I agree here, how does everyone else feels?

> * Game B over Game A. Is not very clear, i think we can explain better the concept. Actually I can't get it... "everything is always discussable"?
Explaining this would be too much information. It will be explained in one of the next content pieces. Game A is the term for the current economy - elbow, egocentric, solely financial profit oriented, zero sum game. Game B is the term for the economy we're trying to build - cooperative, decentralised, self responsible, balance & purpose driven, striving for Nash equilibrium.
> 
> Hope it can help!
> Cheers G.

**Comment by DanM3rcurius** [comment]
> @SecretDecks/founding-core
> How about the following to replace "Love, over fear of change"
> 
> * Love, over fear
> * Fluidity over conformity
> 
> For "Game B over Game A, we may want to link to a video or concise article that can quickly explain this concept to those unfamiliar.

Agree 100%

**Comment by evolayak** [comment]
I like "Fluidity over conformity". And totally agree to link to something or even have a footnote re: Game B over Game A. The artists will be reading this right? If so, I'm not even sure everyone reading it will get the Nash equilibrium reference (not going assume everyone understands game theory or seen a Beautiful Mind). 

Maybe a footnote like:
Game A: Current economic system - egocentric, solely financial profit-oriented, invisible elbow, zero sum game.
Game B: Economic system we're trying to build - cooperative, decentralised, self-responsible, balance and purpose-driven, striving for Nash equilibrium.

**Comment by stephen-rowan** [comment]
Hi everyone - here are some comments on the Mindfulness Policy ..

"We entertain the idea" - I like this phrasing as it suggests a non-dogmatic approach - these ideas are to be worked with and tested etc.

"zero sum games" - from a game theory perspective a zero sum game _is_ one form of a Nash equilibrium. There are many forms of Nash equilibrium that represent different _non-cooperative games_ - precisely because they are non-cooperative they have been used to model "trustless" systems. I criticized non-zero sum outcomes on the Cardano forum on 28th March ([https://forum.cardano.org/t/game-theory-non-zero-sum-outcomes/55371](link)). This is an interesting issue in whether it is possible to build a "trustless" governance blockchain.

"Complementing, over construing" - to construe as in to pre-judge ? I would use the antonym "contest" - e.g. "Complementing, over contesting the environment and individual." 

"Love, over fear of change." - an echo here of 1 Corinthians 13.8 "Love never ends" ? The Tarot Card "The Lovers" has echoes of Plato's Symposium which ranges over the different connotations of love. It might be an interesting exercise to "translate" the Mindfulness Policy into Tarot Arcana.

"Solutioning, over discussing problems." - I like this approach. A pedagogical approach to ethics might use "problem setting" (using thought experiments) to arrive at a solution.

"the only obligation" - In my view the Mindfulness policy needs to be an active document. @wlstylr and I were discussing a possible way to do this with onboarding [here](https://github.com/SecretDecks/Documentation/issues/13#issuecomment-849349756). I would rephrase to something like "Members are obliged to engage with the Mindfulness policy etc ...". This engagement would be iterative and creative.

S

**Comment by wlstylr** [comment]
I just want to touch on "complementing, over construing."  In this context I interpret "construing"  not as directly contesting, but as using existing contribution for personal gain by skewing it in a way where the ethos behind the original contribution is lost.  I prefer contesting.  At least when something is contested the message and meaning is clear.

**Comment by stephen-rowan** [comment]
Yes you are right a construal is not a prejudgement but a "skewing" or interpretation for personal ends. Whereas "Complementing" infers co-operation.

**Comment by DanM3rcurius** [comment]
> I like "Fluidity over conformity". And totally agree to link to something or even have a footnote re: Game B over Game A. The artists will be reading this right? If so, I'm not even sure everyone reading it will get the Nash equilibrium reference (not going assume everyone understands game theory or seen a Beautiful Mind).
> 

Thats why Game B over Game A is much more precise, if the terms are understood. For this I'd like to ask everyone to take a couple of minutes of their time and get to know the terms [Game A](https://www.gameb.wiki/index.php?title=Game_A)  and [Game B](https://www.gameb.wiki/index.php?title=Game_B) from the [Game B wiki](https://www.gameb.wiki/)

- Game A is primarily characterized by scarcity and thus rivalrous or win-lose dynamics:
- Game B is the flag on the hill for an omni-win civilization that maximizes human flourishing

> Maybe a footnote like:
> Game A: Current economic system - egocentric, solely financial profit-oriented, invisible elbow, zero sum game.
> Game B: Economic system we're trying to build - cooperative, decentralised, self-responsible, balance and purpose-driven, striving for Nash equilibrium.

**I'd propose the above footnote with a link to the Game B wiki**

**Comment by DanM3rcurius** [comment]
> Hi everyone - here are some comments on the Mindfulness Policy ..
> 
> "We entertain the idea" - I like this phrasing as it suggests a non-dogmatic approach - these ideas are to be worked with and tested etc.

<3

> 
> "zero sum games" - from a game theory perspective a zero sum game _is_ one form of a Nash equilibrium. There are many forms of Nash equilibrium that represent different _non-cooperative games_ - precisely because they are non-cooperative they have been used to model "trustless" systems. I criticized non-zero sum outcomes on the Cardano forum on 28th March ([https://forum.cardano.org/t/game-theory-non-zero-sum-outcomes/55371](link)). This is an interesting issue in whether it is possible to build a "trustless" governance blockchain.
> 

**Lets leave out Nash equilibrium and use the term *omni win civilisation/economy***

> "Complementing, over construing" - to construe as in to pre-judge ? I would use the antonym "contest" - e.g. "Complementing, over contesting the environment and individual."

Sorry that might have been my inner German-English translation issue. I wanted the term to mean more sth like "obstructing" as in focusing on the differences as weaknesses than as strengths from others.
I feel a healthy amount of contest is necessary for finding good complementations, so I'm unsure about using "contest" in this context; or am I mistaken?

> 
> "Love, over fear of change." - an echo here of 1 Corinthians 13.8 "Love never ends" ? The Tarot Card "The Lovers" has echoes of Plato's Symposium which ranges over the different connotations of love. It might be an interesting exercise to "translate" the Mindfulness Policy into Tarot Arcana.
> 

Ideally we can fashion the journey of secret decks / the autonomous organisation as a major arcana deck, since the 22 majors represent the heroes journey split in the two parts (day & night) of said heroes journey.

> "Solutioning, over discussing problems." - I like this approach. A pedagogical approach to ethics might use "problem setting" (using thought experiments) to arrive at a solution.
> 

Yes! Also Stafford Beers Team Syntegrity Events could enrich this autonomous organisation framework's problem solutioning style.

> "the only obligation" - In my view the Mindfulness policy needs to be an active document. @wlstylr and I were discussing a possible way to do this with onboarding [here](https://github.com/SecretDecks/Documentation/issues/13#issuecomment-849349756). I would rephrase to something like "Members are obliged to engage with the Mindfulness policy etc ...". This engagement would be iterative and creative.

Agree, however what if: New members propose a change to the policy that's not being approved by older members (for whatever hypothetical reason). Now new members are upset and the network could split. From my point of view this mindfulness policy should only be changed after the primer milestone, if the core product/team/soul of the organisation changes as well. Thoughts?  

D.

**Comment by wlstylr** [comment]
Perhaps engaging with the Mindfulness policy is simply sharing a thought or perspective in the form of a comment like we are doing here?  Not a call to amend it, but a comment illustrating engagement and understanding from a new person's perspective.  

I agree that the mindfulness policy shouldn't be something easily modified, and similar to your suggestion that change should only be considered after the primer milestone, what are thoughts on expanding this into part of a repeating milestone completion retrospective process?  After each milestone is completed, there could be an opportunity to review and modify the mindfulness policy based on learnings acquired.  Thoughts?

**Comment by stephen-rowan** [comment]
> Sorry that might have been my inner German-English translation issue. I wanted the term to mean more sth like "obstructing" as in focusing on the differences as weaknesses than as strengths from others.
> I feel a healthy amount of contest is necessary for finding good complementations, so I'm unsure about using "contest" in this context; or am I mistaken?

Yes you are right, @DanM3rcurius - "contest" may be inadequate to express this point - particularly as there can be a positive side to "contesting". It could be rewritten as "Complement rather than obstruct ..." ? or more explicitly as "We complement each other by appreciating rather than obstructing our differences.".

**Comment by stephen-rowan** [comment]
> Perhaps engaging with the Mindfulness policy is simply sharing a thought or perspective in the form of a comment like we are doing here? Not a call to amend it, but a comment illustrating engagement and understanding from a new person's perspective.

Totally agree with this. I am beginning to think that a good GitHub process for this might be to just keep a Discussion Thread going on key policies (e.g. Mindfulness Policy, Contribution policy, Code of Conduct etc.) . That provides a place to engage, expand and capture understanding of the policy- which can then be referred back to when the community decide to amend it.

**Comment by DanM3rcurius** [comment]
> Perhaps engaging with the Mindfulness policy is simply sharing a thought or perspective in the form of a comment like we are doing here? Not a call to amend it, but a comment illustrating engagement and understanding from a new person's perspective.

Yes! Great thinking @wlstylr 🙏🏿 After the primer milestone amendments or improvement proposals should be gathered in comments within discussions.

> 
> I agree that the mindfulness policy shouldn't be something easily modified, and similar to your suggestion that change should only be considered after the primer milestone, what are thoughts on expanding this into part of a repeating milestone completion retrospective process? After each milestone is completed, there could be an opportunity to review and modify the mindfulness policy based on learnings acquired. Thoughts?

YES! I love the idea to integrate the MP into milestone reviews. how about the others? @SecretDecks/founding-core

**Comment by DanM3rcurius** [comment]
> > Perhaps engaging with the Mindfulness policy is simply sharing a thought or perspective in the form of a comment like we are doing here? Not a call to amend it, but a comment illustrating engagement and understanding from a new person's perspective.
> 
> Totally agree with this. I am beginning to think that a good GitHub process for this might be to just keep a Discussion Thread going on key policies (e.g. Mindfulness Policy, Contribution policy, Code of Conduct etc.) . That provides a place to engage, expand and capture understanding of the policy- which can then be referred back to when the community decide to amend it.

Let's do it like this for now. So I'll keep this discussion open but "close" the document prior to finishing the primer milestone.
What about using Gitbook? @stephen-rowan ?

**Comment by DanM3rcurius** [comment]
I put the ♾️ symbol in front of the discussion name to signal this is an ongoing discussion

**Comment by stephen-rowan** [comment]
Replying to https://github.com/SecretDecks/Documentation/issues/10#issuecomment-860590777

@DanM3rcurius

For key documents like Mindfulness Policy (or a Code of Conduct, Contribute.md etc,) - I think it is better to keep them as "In Progress" Issue discussions. Because the issue discussion thread is more fully featured on GitHub (fully versioned with project board status recorded etc.).

Once an Issue is converted to a discussion all that version history is lost. So discussions are best reserved for documentation, drafts or conversations.

A lot of finalized documentation could be migrated to GitBook and may be more user friendly. But once stuff is in GitBook it is "production" and not really easy to review or revert back. GitHub also has a basic wiki function.

**Comment by jkochevar** [comment]
@DanM3rcurius 

Everyone has a piece of the truth. Peace comes in pieces.
